code/modules/admin/verbs/modifyvariables.dm

```diff
@@ -5,7 +5,7 @@ var/list/forbidden_varedit_object_types = list(
 										/datum/admin_rank					//editing my own rank? it's more likely than you think
 									)
 
-var/list/VVlocked = list("vars", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y")
+var/list/VVlocked = list("vars", "client", "virus", "viruses", "cuffed", "last_eaten", "unlock_content", "step_x", "step_y", "force_ending")

```

Comment:
that cuffed one isn't handcuffed, and if it is, it's wrong.
Source; Edited a xenomorph's cuffs off the other day because resist literally wouldn't do it.